public/components/trace_analytics/home.tsx

```diff
@@ -364,6 +366,7 @@ export const Home = (props: HomeProps) => {
             />
           )}
         />
+        <Route path="/" render={() => <Redirect to={defaultRoute} />} />
```

Comment:
why this change?

Reply:
Because we need to remove in-app navigation when the new navigation is enabled. In such case, different apps registered to new left navigation will have difference landing pages.